menus/sgd/main.cfg

```diff
@@ -19,8 +19,8 @@
 set sg2d_directory="${config_directory}"
 export sg2d_directory
 
+loadfont unicode
```

Comment:
Is this supposed to be:
```
loadfont unifont
```

?

Reply:
No. I guess the word "font" would be redundant inside a "fonts" directory. [This GRUB unit test](http://git.savannah.gnu.org/cgit/grub.git/diff/tests/grub_func_test.in?id=4f47d809c7b2637fde651abb43352b34d1491f7c) makes it seem like it's the new default, rather than Ubuntu being excentric.